static/js/stream_popover.js

```diff
@@ -372,9 +372,12 @@ exports.register_click_handlers = function () {
         build_starred_messages_popover,
     );
 
-    $("body").on("click", ".move-topic-dropdown .list_item", (e) => {
+    $("body").on("click keypress", ".move-topic-dropdown .list_item", (e) => {
         e.preventDefault();
 
+        if (e.type === "keypress" && e.which !== 13) {
+            return;
+        }
```

Comment:
This change seems to be not related to the "overflow" description.

Reply:
Ahh, Apologies for these silly commit errors!